src/mpi/comm/commutil.c

```diff
@@ -340,6 +361,10 @@ int MPIR_Comm_commit(MPIR_Comm * comm)
     int num_local = -1, num_external = -1;
     int local_rank = -1, external_rank = -1;
     int *local_procs = NULL, *external_procs = NULL;
+    int max_node_id, node_id, i, *nodemap = NULL;
+    FILE *fp;
+    char str[9999], str1[9999];
```

Comment:
We probably want to replace `9999` with a macro or constant here so it is more readable and  can potentially configure it

Reply:
I agree